firstDirectory/Inheritance/Vehicle.java

```diff
@@ -0,0 +1,21 @@
+package firstDirectory.Inheritance;
+
+public class Vehicle {
+
+    String color = "blue";
+    int numberOfWheels = 4;
+
```

Comment:
how about passing this class variable as constructor parameters? that would be a more dynamic and close to real life approch

Reply:
Thank you I think I understood.